DAOS-16549 test: pool/create_all_hw.py - Use storage: auto

shimizukko opened the pull request:
Update pool/create.py test yaml to use storage: auto

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all
### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate owners.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

shimizukko committed:
DAOS-16549 test: pool/create.py - Use storage: auto

Update pool/create.py test yaml to use storage: auto

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all
Signed-off-by: Makito Kano <makito.kano@intel.com>

shimizukko committed:
DAOS-16549 test: Comment out 1 and 2-target variants becaue they are failing due to a bug

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all
Signed-off-by: Makito Kano <makito.kano@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-15709/1/execution/node/1037/log

daosbuild1 commented:
Test stage Functional Hardware Medium MD on SSD completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-15709/1/execution/node/1053/log

shimizukko committed:
Merge branch 'master' into makito/DAOS-16549

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all

shimizukko committed:
DAOS-16549 test: Update launch_utils.py and storage_utils.py to use storage: auto with !mux

In launch_utils.py, obtain field under server_configurations.
e.g., server_config_1_target.

In storage_utils.py, create extra yaml based on the test yaml.

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all
Signed-off-by: Makito Kano <makito.kano@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Medium MD on SSD completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-15709/2/execution/node/1006/log

shimizukko committed:
DAOS-16549 test: Fixed control_metadata indentation

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all
Signed-off-by: Makito Kano <makito.kano@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Medium MD on SSD completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-15709/3/execution/node/1006/log

shimizukko committed:
Merge branch 'master' into makito/DAOS-16549

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all

daosbuild1 commented:
Test stage Functional Hardware Medium MD on SSD completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-15709/4/execution/node/1037/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status UNSTABLE.  https://build.hpdd.intel.com/job/daos-stack/job/daos//view/change-requests/job/PR-15709/4/testReport/

shimizukko committed:
Merge branch 'master' into makito/DAOS-16549

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all

shimizukko committed:
DAOS-16549 test: Update test yaml to mux targets using !filter-only

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all
Signed-off-by: Makito Kano <makito.kano@intel.com>

shimizukko committed:
DAOS-16549 test: Revert changes in launch_utils.py and storage_utils.py

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all
Signed-off-by: Makito Kano <makito.kano@intel.com>

shimizukko committed:
DAOS-16549 test: Fix pylint

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all
Signed-off-by: Makito Kano <makito.kano@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Medium MD on SSD completed with status UNSTABLE.  https://build.hpdd.intel.com/job/daos-stack/job/daos//view/change-requests/job/PR-15709/6/testReport/

shimizukko committed:
DAOS-16549 test: Add 1 target and 2 target test variants

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all
Signed-off-by: Makito Kano <makito.kano@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status UNSTABLE.  https://build.hpdd.intel.com/job/daos-stack/job/daos//view/change-requests/job/PR-15709/7/testReport/

daosbuild1 commented:
Test stage Functional Hardware Medium MD on SSD completed with status UNSTABLE.  https://build.hpdd.intel.com/job/daos-stack/job/daos//view/change-requests/job/PR-15709/7/testReport/

shimizukko committed:
Merge branch 'master' into makito/DAOS-16549

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all

shimizukko committed:
DAOS-16549 test: Remove debug pool create and query

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all
Signed-off-by: Makito Kano <makito.kano@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Medium MD on SSD completed with status UNSTABLE.  https://build.hpdd.intel.com/job/daos-stack/job/daos//view/change-requests/job/PR-15709/8/testReport/

shimizukko committed:
Merge branch 'master' into makito/DAOS-16549

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all

daosbuild1 commented:
Test stage Functional Hardware Medium MD on SSD completed with status UNSTABLE.  https://build.hpdd.intel.com/job/daos-stack/job/daos//view/change-requests/job/PR-15709/9/testReport/

shimizukko committed:
Merge branch 'master' into makito/DAOS-16549

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all

shimizukko committed:
DAOS-16549 test: Use TRACE log level for control plane to check available space calculation

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all
Signed-off-by: Makito Kano <makito.kano@hpe.com>

daosbuild1 commented:
Test stage Functional Hardware Medium MD on SSD completed with status UNSTABLE.  https://build.hpdd.intel.com/job/daos-stack/job/daos//view/change-requests/job/PR-15709/10/testReport/

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-15709/10/execution/node/1053/log

shimizukko committed:
Merge branch 'master' into makito/DAOS-16549

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all

daosbuild1 commented:
Test stage Functional Hardware Medium MD on SSD completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-15709/11/execution/node/967/log

shimizukko committed:
Merge branch 'master' into makito/DAOS-16549

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: pool_create_all

daosbuild1 commented:
Test stage Functional Hardware Medium MD on SSD completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-15709/12/execution/node/940/log
